### PR TITLE
[IOTDB-1385] Extract the super user and password to the config

### PR DESCRIPTION
**Pull request by wangchao316**
In some scenarios, the super user needs to be specified based on the environment.  Extract the super user and password to the config

### CODE REVIEW

**Comment by wangchao316** [comment]
@LebronAl @neuyilan @mychaow hi, could you please review?
Thanks

**Review by neuyilan — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java [hunk 1]
```diff
@@ -701,19 +701,23 @@ private void loadProps() {
       }
 
       conf.setAuthorizerProvider(
-          properties.getProperty(
-              "authorizer_provider_class",
-              "org.apache.iotdb.db.auth.authorizer.LocalFileAuthorizer"));
+          properties.getProperty("authorizer_provider_class", conf.getAuthorizerProvider()));
       // if using org.apache.iotdb.db.auth.authorizer.OpenIdAuthorizer, openID_url is needed.
-      conf.setOpenIdProviderUrl(properties.getProperty("openID_url", ""));
+      conf.setOpenIdProviderUrl(properties.getProperty("openID_url", conf.getOpenIdProviderUrl()));
```
Comment (LebronAl) [inline server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java:706]: The default value for `openIdProviderUrl` is `null`, which is different with `""`.
It seems that here will throw an exception:
image

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java [hunk 2]
```diff
@@ -701,19 +701,23 @@ private void loadProps() {
...
 
       conf.setEnablePartition(
           Boolean.parseBoolean(
-              properties.getProperty("enable_partition", conf.isEnablePartition() + "")));
+              properties.getProperty(
```
Comment (LebronAl) [inline server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java:710]: Hi, As we set these field here, it's better to remove commented code in 599-611 lines

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 3]
```diff
@@ -831,3 +831,9 @@ timestamp_precision=ms
 # time range for partitioning data inside each storage group, the unit is second
 # Datatype: long
 # partition_interval=604800
+
+# admin username, default is root
+admin_name=root
```
Comment (LebronAl) [inline server/src/assembly/resources/conf/iotdb-engine.properties:836]: We'd better keep these comments in a consistent format.
Such as 
```
# admin username, default is root
# Datatype: string
# admin_name=root
```

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 4]
```diff
@@ -831,3 +831,9 @@ timestamp_precision=ms
...
+
+# admin password, default is root
+admin_password=root
```
Comment (LebronAl) [inline server/src/assembly/resources/conf/iotdb-engine.properties:839]: same as above

**Review by LebronAl — COMMENTED** [review]
Great work！
Only minor issues, PTAL.

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java (hunk 1 above)
Comment (wangchao316) [reply server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java:692]: Thanks xinyu
do it.

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java (hunk 2 above)
Comment (wangchao316) [reply server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java:696]: done

File: server/src/assembly/resources/conf/iotdb-engine.properties (hunk 3 above)
Comment (wangchao316) [reply server/src/assembly/resources/conf/iotdb-engine.properties]: done

File: server/src/assembly/resources/conf/iotdb-engine.properties (hunk 4 above)
Comment (wangchao316) [reply server/src/assembly/resources/conf/iotdb-engine.properties]: done

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 5]
```diff
@@ -831,3 +831,9 @@ timestamp_precision=ms
 # time range for partitioning data inside each storage group, the unit is second
 # Datatype: long
 # partition_interval=604800
+
+# admin username, default is root
+#admin_name=root
```
Comment (LebronAl) [inline server/src/assembly/resources/conf/iotdb-engine.properties:836]:
```suggestion
# admin username, default is root
# Datatype: string
# admin_name=root
```

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 6]
```diff
@@ -831,3 +831,9 @@ timestamp_precision=ms
...
+
+# admin password, default is root
+#admin_password=root
```
Comment (LebronAl) [inline server/src/assembly/resources/conf/iotdb-engine.properties:839]:
```suggestion
# admin password, default is root
# Datatype: string
# admin_password=root
```

**Review by LebronAl — APPROVED** [review]
LGTM

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 7]
```diff
@@ -831,3 +831,13 @@ timestamp_precision=ms
 # time range for partitioning data inside each storage group, the unit is second
 # Datatype: long
 # partition_interval=604800
+
+# admin username, default is root
```
Comment (LebronAl) [inline server/src/assembly/resources/conf/iotdb-engine.properties:835]:
```suggestion
```

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 8]
```diff
@@ -831,3 +831,13 @@ timestamp_precision=ms
...
+# admin username, default is root
+# Datatype: string
+# admin_name=root
+
+# admin password, default is root
```
Comment (LebronAl) [inline server/src/assembly/resources/conf/iotdb-engine.properties:840]:
```suggestion
```

**Review by LebronAl — APPROVED** [review]

**Pull request merged**